torchani/aev.py

```diff
@@ -2,19 +2,18 @@
 import torch
 from . import _six  # noqa:F401
 import math
-from typing import Tuple
+from torch import Tensor
+from typing import Tuple, Optional
 
 
-# @torch.jit.script
 def cutoff_cosine(distances, cutoff):
-    # type: (torch.Tensor, float) -> torch.Tensor
+    # type: (Tensor, float) -> Tensor
     # assuming all elements in distances are smaller than cutoff
     return 0.5 * torch.cos(distances * (math.pi / cutoff)) + 0.5
 
 
-# @torch.jit.script
 def radial_terms(Rcr, EtaR, ShfR, distances):
-    # type: (float, torch.Tensor, torch.Tensor, torch.Tensor) -> torch.Tensor
+    # type: (float, float, float, Tensor) -> Tensor
```

Comment:
Shouldn't `EtaR` and `ShfR` be `Tensor`?

Reply:
Only `Rcr` and `Rca` are floats. The rest are just tensors. Some, with one element (`EtaR`, `ShfR` and `EtaA`)

Reply:
So shouldn't the type hint be `# type: (float, Tensor, Tensor, Tensor) -> Tensor`

Reply:
Resolved in 92584cc